### ISSUE TITLE
Break pure-render checks

### ISSUE THREAD

**Issue by necolas** [rackt/react-a11y#77]
This module generates new a11y id's every render, meaning that pure render mixin doesn't work.

**Comment by blairanderson** [comment]
Why is this important for testing accessibility?

**Comment by globexdesigns** [comment]
It's an issue because during development performance of the application is extremely poor with react-a11y enabled. Ideally we'd like to keep react-a11y enabled always so that new warnings are immediately visible to developers.

**Comment by necolas** [comment]
Yeah it's important because it slows iteration speed in development and makes it hard to identify similar performance regressions not caused by react-a11y.
